src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx

```diff
@@ -161,7 +161,7 @@ export default function DelegatedSupportCampsUI({
               Are you sure, you want to remove your delegate support given to
               pranav_telentelgia under the topic{" "}
```

Comment:
`pranav_talentelgia` is it a typo? or we need it

Reply:
This is static content. Once APIs are ready it will come from the apis